crates/zombie_actor/src/ghost_tracker.rs

```diff
@@ -3,7 +3,7 @@ use std::collections::HashMap;
 
 use crate::{ghost_error::ErrorKind, GhostError, GhostResult, RequestId, WorkWasDone};
 
-const DEFAULT_TIMEOUT: std::time::Duration = std::time::Duration::from_millis(2000);
+const DEFAULT_TIMEOUT: std::time::Duration = std::time::Duration::from_millis(20000);
```

Comment:
Ouch, 20 seconds a long time to make sure a test passes- do you know which one was sensitive to this timing?

Reply:
Added TODO